src/components/app/MenuButtons/MetaInfo.css

```diff
@@ -14,35 +13,15 @@
 
   /* Let this box shrink down to the size of the three dots and a bit of text. */
   min-width: 75px;
+  flex-flow: column nowrap;
 
   /* Shrink down twice as fast as the range selection labels. */
   flex-shrink: 2;
   padding: 0;
 }
```

Comment:
I think we need a left border also, so that it's clear that it's a clickable button.

Reply:
Yes, I will apply same border as `Docs...` has.